src/vision/roadmap.md

```diff
@@ -69,7 +69,7 @@ Clicking on active initiatives also shows a list of *milestones*. These mileston
 [fundamentals]: https://rust-lang.github.io/async-fundamentals-initiative/
 [Async closures]: https://rust-lang.github.io/async-fundamentals-initiative/design-discussions/async_closures.html
 [Boxable async functions]: ./roadmap/async_fn/boxable.md
-[Async main and tests]: ./roadmap/async_fn/async_main_and_tests.md
+[Async main and tests]: https://rust-lang.github.io/wg-async
```

Comment:
This doesn't look like the right link?

Reply:
Yeah sorry, I may copy and paste the wrong link, it is this one https://vincenzopalazzo.github.io/async-main-and-tests-initiative/index.html and now should be included inside the change